docs/en/04.venus-sector-manager-config.md

```diff
@@ -0,0 +1,585 @@
+#GasOverEstimation = 1.2
+#MaxFeeCap = "5 nanoFIL"
+#MessageConfidential = 15
+#ReleaseCondidential = 30
+[Miners.SnapUp.Retry]
+#MaxAttempts = 10
+#PollInterval = "3m0s"
+#APIFailureWait = "3m0s"
+#LocalFailureWait = "3m0s"
+[Miners.Commitment]
+#Confidence = 10
+[Miners.Commitment.Pre]
+#SendFund = true
+#Sender = "t1abjxfbp274xpdqcpuaykwkfb43omjotacm2p3za"
+#GasOverEstimation = 1.2
+#MaxFeeCap = "5 nanoFIL"
+[Miners.Commitment.Pre.Batch]
+#Enabled = false
+#Threshold = 16
+#MaxWait = "1h0m0s"
+#CheckInterval = "1m0s"
+#GasOverEstimation = 1.2
+#MaxFeeCap = "5 nanoFIL"
+[Miners.Commitment.Prove]
+#SendFund = true
+#Sender = "t1abjxfbp274xpdqcpuaykwkfb43omjotacm2p3za"
+#GasOverEstimation = 1.2
+#MaxFeeCap = "5 nanoFIL"
+[Miners.Commitment.Prove.Batch]
+#Enabled = false
+#Threshold = 16
+#MaxWait = "1h0m0s"
+#CheckInterval = "1m0s"
+#GasOverEstimation = 1.2
+#MaxFeeCap = "5 nanoFIL"
+[Miners.Commitment.Terminate]
+#Sender = "t1abjxfbp274xpdqcpuaykwkfb43omjotacm2p3za"
+#SendFund = true
+#GasOverEstimation = 1.2
+#MaxFeeCap = "5 nanoFIL"
+[Miners.Commitment.Terminate.Batch]
+#Enabled = false
+#Threshold = 5
+#MaxWait = "1h0m0s"
+#CheckInterval = "1m0s"
+#GasOverEstimation = 1.2
+#MaxFeeCap = "5 nanoFIL"
+[Miners.Post]
+#Sender = "t1abjxfbp274xpdqcpuaykwkfb43omjotacm2p3za"
+#Enabled = true
+#StrictCheck = true
+#GasOverEstimation = 1.2
+#MaxFeeCap = "5 nanoFIL"
+#Confidence = 10
+#ChallengeConfidence = 10
+#MaxPartitionsPerPoStMessage = 0
+#MaxPartitionsPerRecoveryMessage = 0
+[Miners.Proof]
+#Enabled = false
+#
+````
+
+We will break down each configurable item one by one.
+
+
+## [Common]
+
+`Common` section includes common configuration, which is further divided into three sub-configuration items:
+
+
+
+### [Common.API]
+
+`Common.API` is interface related configuration, its content includes:
+
+````
+[Common.API]
+# Chain service address, required, string type
+# Fill in according to the actual situation of the service used
+Chain = "/ip4/{api_host}/tcp/{api_port}"
+
+# Message service address, required, string type
+# Fill in according to the actual situation of the service used
+Messager = "/ip4/{api_host}/tcp/{api_port}"
+
+# Market service address, required, string type
+# Fill in according to the actual situation of the service used
+Market = "/ip4/{api_host}/tcp/{api_port}"
+
+# Event gateway service address, required, string type
+# Fill in according to the actual situation of the service used
+Gateway = ["/ip4/{api_host}/tcp/{api_port}"]
+
+# service token, required, string type
+# Fill in according to the actual situation of the service used
+Token = "{some token}"
+
+# Interval time for detecting chain height changes, optional, time type
+# Default is 1min
+#ChainEventInterval = "1m0s"
+````
+
+
+
+### [[Common.PieceStores]]
+
+`Common.PieceStores` is used for configuring local deal `piece` data. When there is available offline deal, you can configure this item to avoid getting the deal `piece` data through the public network.
```

Comment:
```suggestion
`Common.PieceStores` is used for configuring local deal `piece` data. When there is available offline deal, you can configure this item to avoid getting the deal `piece` data through public network traffic.
```